LBHFSSPublicAPI/terraform/production/main.tf

```diff
@@ -58,7 +58,7 @@ module "postgres_db_production" {
   db_identifier = "fss-public-dev-db"
   db_instance_class = "db.t2.micro"
   db_name = "fss-public_dev"
```

Comment:
Sorry I didn't notice this before.  The environment suffix '_dev' needs to be updated to reflect the environment.